__tests__/app.test.js

```diff
@@ -47,4 +47,43 @@ describe('testing GET /api', () => {
             expect(allEndpoints).toEqual(response._body);
         })
     });
+});
+
+describe('testing GET /api/articles/:article_id', () => {
+    test('should respond with a 200 status code', () => {
+        return request(app).get("/api/articles/1")
+        .expect(200);
+    });
+    test('should respond with an article object based on the given id', () => { 
+        const expected = {article : [{   
+            article_id: 2,
```

Comment:
Its important here to be specific about which article id should be returned - well done✅